src/php/ext/grpc/channel.c

```diff
@@ -148,37 +148,33 @@ PHP_METHOD(Channel, __construct) {
                          "Channel expects a string and an array", 1 TSRMLS_CC);
     return;
   }
-  if (args_array == NULL) {
```

Comment:
Yes all caller to the `Channel` constructor - the array is required. Updated the few lines up. The null case should be unreachable.